### PR TITLE
Add Java role

### PR DESCRIPTION
**Pull request by PhilipGough**
ping @odra

### CODE REVIEW

**Comment by odra** [comment]
👀

File: java/meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+galaxy_info:
+  author: AeroGear
+  company: AeroGear
+  description: "Java for Red Hat family OS"
+  issue_tracker_url: https://issues.jboss.org/projects/AGDIGGER/issues
+  license: "license (Apache 2)"
+  min_ansible_version: "2.2.0"
+  platforms:
+    -
+      name: fedora
```
Comment (odra) [inline java/meta/main.yml]: should we add "centos" as well?

**Comment by odra** [comment]
Verified by running:

```
ansible-playbook -i sample-inventory.cfg sample-build-playbook.yml --tags=install-java
```

LGTM 👍

File: java/meta/main.yml (hunk 1 above)
Comment (PhilipGough) [inline java/meta/main.yml]: Will do

Comment (PhilipGough) [inline java/meta/main.yml]: Centos doesn't exist in the list of supported  platforms See https://galaxy.ansible.com/api/v1/platforms/. Lets leave as is

**Pull request merged**